lib/fast-path.js

```diff
@@ -13,23 +13,23 @@ const FPp = FastPath.prototype;
 module.exports = FastPath;
 
 // Static convenience function for coercing a value to a FastPath.
-FastPath.from = function(obj) {
+FastPath.from = function (obj) {
   return obj instanceof FastPath
     // Return a defensive copy of any existing FastPath instances.
     ? obj.copy()
     // Otherwise use obj as the value of the new FastPath instance.
     : new FastPath(obj);
 };
 
-FPp.copy = function copy() {
-  const copy = Object.create(FastPath.prototype);
-  copy.stack = this.stack.slice(0);
-  return copy;
+FPp.copy = function () {
+  const cp = Object.create(FastPath.prototype);
+  cp.stack = this.stack.slice(0);
+  return cp;
```

Comment:
It could but in another PR after this I'm removing some dead code (which `FFp.copy` will become) so what it's called won't matter too much in the next PR.